README.md

```diff
@@ -1,8 +1,148 @@
-# Phootwork library #
+# Phootwork library
 
 [![Build Status](https://travis-ci.org/phootwork/phootwork.svg?branch=master)](https://travis-ci.org/phootwork/phootwork)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/phootwork/phootwork/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/phootwork/phootwork/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/phootwork/phootwork/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/phootwork/phootwork/?branch=master)
 [![License](https://img.shields.io/github/license/phootwork/phootwork.svg?style=flat-square)](https://packagist.org/packages/phootwork/phootwork)
 
-The phootwork php library fills gaps in the php language and provides better solutions than the existing ones php offers.
+Phootwork is a collection of php libraries which fill gaps in the php language and provides consistent object oriented solutions
+where the language natively offers only functions.
+
+The Phootwork package includes:
+
+- [collection](https://github.com/phootwork/collection) a library to model several flavours of collections
+- [file](https://github.com/phootwork/file) an object oriented library to manipulate filesystems elements (*stream compatible*)
+- [json](https://github.com/phootwork/json) a json library, with clean syntax and proper error handling
+- [lang](https://github.com/phootwork/lang) a library to manipulate arrays and strings in an object oriented way
+- [tokenizer](https://github.com/phootwork/tokenizer) an easy to use tokenizer library for PHP code
+- [xml](https://github.com/phootwork/xml) an object oriented xml utility library
+
+## Installation
+
+We use [composer](https://getcomposer.org) as dependency manager and distribution system. To install the library run:
+
+```bash
+composer require phootwork/phootwork
+```
+
+Each single package can be installed separately. I.e. if you want to include in your project the `collection` library only:
+
+```bash
+composer require phootwork/collection
+```
+
+> **Note**: the single library packages does not ship with tests and --dev dependencies. If you want to run the test suite or
+> contribute to the library, you have to install the whole `phootwork/phootwork` package.
+
+## A little taste
+
+The following examples show what you can find in this library. You can discover much, much more by reading the documentation
+and the api.
+
+### A little taste of **lang** library (`phootwork\lang\Text` class);
```

Comment:
Same here: `A Little Taste of **lang** Library ...`

Reply:
Done